packages/knip/test/cli-reporter-markdown.test.ts

```diff
@@ -0,0 +1,31 @@
+import assert from 'node:assert/strict';
+import test from 'node:test';
+import { resolve } from '../src/util/path.js';
+import { execFactory } from './helpers/exec.js';
+
+const cwd = resolve('fixtures/module-resolution-non-std');
+
+const exec = execFactory(cwd);
+
+test('knip --reporter markdown', () => {
+  const markdown = `## Unused files (1)
```

Comment:
Maybe it would be good to have one document title (e.g. `# Knip Report` or something like that) with the issue types being second-level headings?

Reply:
Yeah makes sense